neotools/commands.py

```diff
@@ -55,14 +55,14 @@ def read_all_files(applet_id, path, name_format):
         applet_id = AppletIds.ALPHAWORD
     with Device.connect() as device:
         files = file.list_files(device, applet_id)
-        for file_attrs in files:
-            text = read_text(device, applet_id, file_attrs)
+        for file_index, file_attrs in enumerate(files, start=1):
```

Comment:
Would the index shift if alphaword has its first files cleared?! It is worth checking this case.